lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/NetconfNmdaBaseServiceImpl.java

```diff
@@ -0,0 +1,238 @@
+import org.opendaylight.yang.gen.v1.urn.tbd.params.xml.ns.yang.network.topology.rev131021.NodeId;
+import org.opendaylight.yangtools.rfc7952.data.api.NormalizedMetadata;
+import org.opendaylight.yangtools.rfc7952.data.util.ImmutableMetadataNormalizedAnydata;
+import org.opendaylight.yangtools.rfc7952.data.util.ImmutableNormalizedMetadata;
+import org.opendaylight.yangtools.yang.common.Empty;
+import org.opendaylight.yangtools.yang.common.QName;
+import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier;
+import org.opendaylight.yangtools.yang.data.api.schema.AnydataNode;
+import org.opendaylight.yangtools.yang.data.api.schema.ChoiceNode;
+import org.opendaylight.yangtools.yang.data.api.schema.DataContainerChild;
+import org.opendaylight.yangtools.yang.data.api.schema.LeafSetEntryNode;
+import org.opendaylight.yangtools.yang.data.api.schema.NormalizedAnydata;
+import org.opendaylight.yangtools.yang.data.api.schema.NormalizedNode;
+import org.opendaylight.yangtools.yang.data.impl.schema.Builders;
+import org.opendaylight.yangtools.yang.data.impl.schema.ImmutableNodes;
+import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableAnydataNodeBuilder;
+import org.opendaylight.yangtools.yang.data.util.ImmutableNormalizedAnydata;
+import org.opendaylight.yangtools.yang.model.api.DataSchemaNode;
+import org.opendaylight.yangtools.yang.model.api.SchemaContext;
+
+public class NetconfNmdaBaseServiceImpl extends NetconfBaseServiceImpl implements NetconfNmdaBaseService {
+
+    public static final QName NETCONF_NMDA_EXTENSION_QNAME = QName.create("urn:ietf:params:xml:ns:yang:ietf-netconf-nmda", "2019-01-07", "ietf-netconf-nmda").intern();
+    public static final QName NETCONF_GET_DATA_QNAME = QName.create(NETCONF_NMDA_EXTENSION_QNAME, "get-data");
+    public static final QName NETCONF_EDIT_DATA_QNAME = QName.create(NETCONF_NMDA_EXTENSION_QNAME, "edit-data").intern();
+    private static final YangInstanceIdentifier.NodeIdentifier NETCONF_FILTER_NODEID =
+            YangInstanceIdentifier.NodeIdentifier.create(QName.create(NETCONF_GET_DATA_QNAME, "subtree-filter"));
+    private static final YangInstanceIdentifier.NodeIdentifier NETCONF_FILTER_CHOICE_NODEID =
+            YangInstanceIdentifier.NodeIdentifier.create(QName.create(NETCONF_GET_DATA_QNAME, "filter-spec"));
+    private static final YangInstanceIdentifier.NodeIdentifier NETCONF_ORIGIN_FILTERS_CHOICE_NODEID =
+            YangInstanceIdentifier.NodeIdentifier.create(QName.create(NETCONF_GET_DATA_QNAME, "origin-filters"));
+    private static final QName NETCONF_OPERATION_QNAME_LEGACY = NETCONF_OPERATION_QNAME.withoutRevision().intern();
+    private static final YangInstanceIdentifier.NodeIdentifier NETCONF_EDIT_DATA_CONFIG_NODEID =
+            YangInstanceIdentifier.NodeIdentifier.create(QName.create(NETCONF_EDIT_DATA_QNAME, "config"));
+
+    private final DOMRpcService domRpcService;
```

Comment:
these two are already present in NetconfBaseServiceImpl -- expose access to them through a getter

Reply:
Please, see the new commit.